### DISCUSSION TITLE
postgresql12 install on AL2023

### DISCUSSION THREAD

**Discussion by cmccabe0728** [amazonlinux/amazon-linux-2022#527] [Q&A]
Is there a way  `postgresql 12` be installed on AL2023? Thank you.

I can install 15 via dnf:

    sudo dnf install postgresql15

but can not seem to run `initdb`.

**Comment by siddudubey** [comment]
Facing same issue.
```
sudo postgresql-setup initdb
sudo: postgresql-setup: command not found
```

**Reply by siddudubey** [reply]
My issue was due to an incomplete installation, I was running `yum install postgres15` which would only install the postgresql15-proivate-libs. Once I properly installed the server using, `yum install postgres15-server`, this worked.
